C/Coin-OR/Bonmin/build_tarballs.jl

```diff
@@ -0,0 +1,63 @@
+using BinaryBuilder, Pkg
+
+name = "Bonmin"
+version = v"1.8.8"
+
+sources = [
+    GitSource("https://github.com/coin-or/Bonmin.git",
+              "65c56cea1e7c40acd9897a2667c11f91d845bb7b"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/Bonmin*
+
+# Remove misleading libtool files
+rm -f ${prefix}/lib/*.la
+update_configure_scripts
+
+# old and custom autoconf
+sed -i s/elf64ppc/elf64lppc/ configure
+
+mkdir build
+cd build/
+
+export CPPFLAGS="${CPPFLAGS} -I${prefix}/include -I$prefix/include/coin"
+export CXXFLAGS="${CXXFLAGS} -std=c++11"
+
+if [[ ${target} == *mingw* ]]; then
+    export LDFLAGS="-L$prefix/bin"
+fi
+
+../configure \
+    --prefix=$prefix \
+    --build=${MACHTYPE} \
+    --host=${target} \
+    --enable-shared \
+    --with-asl-lib="-lasl -lipoptamplinterface"
+
+make
+make install
+"""
+
+platforms = supported_platforms()
+platforms = filter!(!Sys.isfreebsd, platforms)
+platforms = expand_cxxstring_abis(platforms)
+platforms = expand_gfortran_versions(platforms)
+platforms = filter(x -> cxxstring_abi(x) != "cxx03", platforms)
```

Comment:
Also, if we need `-std=c++11`, is there a better way to do the above?

Reply:
What do you mean exactly? C++ standard of the code does _**not**_ have to do with the string ABI: https://docs.binarybuilder.org/stable/building/#Expanding-C-string-ABIs-or-libgfortran-versions

Reply:
Ah. It was giving me warnings so I added this.

Reply:
Ok, now I've seen the warning you're referring to. Filtering out the C++03 string ABI was good

Reply:
> Filtering out the C++03 string ABI was good

To clarify: does this mean I should re-add the filter?

Reply:
Yes please, because we apparently aren't actually building with the C++03 string ABI anyways for some reasons